source/language/pulses.rst

```diff
@@ -72,16 +69,6 @@ the most specific definition found for a given operation. Thus, given,
 the operation ``rx(pi/2) %0`` would match to (3), ``rx(pi) %0`` would
 match (2), ``rx(pi/2) %1`` would match (1).
 
-Users specify the grammar used inside ``defcal`` blocks with a ``defcalgrammar "name"`` declaration, or by
-an optional grammar string in a ``defcal`` definition, e.g.
-
-.. code-block:: c
-
-   defcalgrammar "openpulse";
```

Comment:
Could we follow the qasm grammar? A qasm file could be started with:
```
OPENQASM 3;
OPENPULSE 3;
```
https://qiskit.github.io/openqasm/grammar/index.html

Reply:
I'm not sure about the resolution, but I do think including a version number is important as the pulse grammar (or whatever calibration grammar is used) will likely evolve in time.